app/main.py

```diff
@@ -2,5 +2,19 @@
 
 
 def cache(func: Callable) -> Callable:
```

Comment:
This is a performance issue. The function `cache` is a decorator that stores the results of the decorated function to avoid recalculating the same results. This is a good practice when the decorated function is computationally expensive.